deploy/docs/Best_Practices.md

```diff
@@ -238,6 +240,23 @@ metadata:
               key: k8s_container_name
 ```
 
+### Difference between /var/log/pods and /var/log/containers
```

Comment:
Why this is subsection of "Overriding chart resource names with `fullnameOverride"?
It seems that you should remove one `#`

Reply:
The title of this section is misleading. In this section it is described how `fluentd.tag` is  built and how it can be used, not the difference between /var/log/pods and /var/log/containers

Reply:
This is subsection of `## Collecting logs from /var/log/pods`
Not sure if this should go as separate section

Reply:
Suggestion for the title: Form of `fluentd.tag` for /var/log/pods and /var/log/containers

Reply:
changed to `Fluentd tag for /var/log/pods and /var/log/containers`